app/models/shipment.rb

```diff
@@ -0,0 +1,19 @@
+class Shipment < ApplicationRecord
+	belongs_to :business_user
+	has_many :shipment_products
+	has_many :products, through: :shipment_products
+	accepts_nested_attributes_for :shipment_products, reject_if: :all_blank, allow_destroy: true
+
+	scope :this_month, -> {where(:created_at => (Time.now.beginning_of_month..Time.zone.now))}	
```

Comment:
Just for the sake of uniformity, let's not use hash rockets - so `:created_at =>`  should be `created_at:`